src/Build.UnitTests/BackEnd/TaskHostFactory_Tests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection;
+using Microsoft.Build.Execution;
+using Microsoft.Build.UnitTests;
+using Shouldly;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Build.Engine.UnitTests.BackEnd
+{
+    public sealed class TaskHostFactory_Tests
+    {
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp, "https://github.com/microsoft/msbuild/issues/5158")]
```

Comment:
Does this work on Mono?

Reply:
Does Mono support out-of-proc task hosts? I was going to find out with the automated tests.